src/components/templates/PartyMap/components/Map/MapRoom.tsx

```diff
@@ -44,15 +45,19 @@ export const MapRoom: React.FC<MapRoomProps> = ({
 
   const containerClasses = classNames("maproom", {
     "maproom--unclickable": isUnclickable,
+    "maproom--iframe": isIframe,
     "maproom--always-show-label":
       !isUnclickable &&
+      !isIframe &&
```

Comment:
As per other comments in this review, refactor to use `NON_INTERACTABLE_ROOM_TYPES` (or similar) extracted into `src/settings`:

```suggestion
      !NON_INTERACTABLE_ROOM_TYPES.includes(room.type) &&
```

Reply:
In fact.. since we're going to use this in a few places in this file, it probably makes sense to make it a standalone `const` up near `isUnclickable`/`isIframe`, then reuse that here/elsewhere:

```typescript
const isNonInteractableRoom = NON_INTERACTABLE_ROOM_TYPES.includes(room.type);
```

Then here we would just use:
```suggestion
      !isNonInteractableRoom &&
```

Reply:
It might even be simpler/more obvious to just make it:

```typescript
const isRoomInteractable = !NON_INTERACTABLE_ROOM_TYPES.includes(room.type);
```

(to avoid the 'double negative' everywhere it's used)

Reply:
Since the naming has changed, I don't think it's relevant?

All of the other comments seem to be addressed